examples/snippets/modules/ROOT/examples/akkastreams-grpc-java/src/main/java/sensordata/Logger.java

```diff
@@ -14,7 +14,8 @@
     private final ProtoInlet<SensorData> inlet = new ProtoInlet<SensorData>(
             "in",
             SensorData.class,
-            true
+            true,
+            (inBytes, throwable) -> null
```

Comment:
@michael-read Did you add this just to skip and *not* log?
The Java API has a `ProtoInlet.create` method, though it does not have an option to provide the error handler (in case it is important to you not to log but to skip).

Reply:
There is now a Java API ProtoInlet.create that takes a `errorHandler: (Array[Byte], Throwable) => Optional[T]` would be nice if you used that instead of returning null

Reply:
yes, I was just skipping it. I'm happy to add a log, if it'll make it better.

Reply:
how about something like:
```
            (inBytes, throwable) -&gt; {                context().system().log().error(String.format("an exception occurred on inlet: %s -&gt; (hex string) %x", throwable.getMessage(), inBytes));                return null; // skip the element            });
            ```

Reply:
how about now @RayRoestenburg ?

Reply:
Sorry missed this comment.
Can you return an `Optional` instead and use the Java API (`ProtoInlet.create`)?

Reply:
I can't see how I can use `java.util.Optional` since this is scala.Option. So best I can figure is 
```
    private final ProtoInlet<SensorData> inlet = (ProtoInlet<SensorData>) ProtoInlet.create("in", SensorData.class)
            .withErrorHandler((inBytes, throwable) -> {
                context().system().log().error(String.format("an exception occurred on inlet: %s -> (hex string) %h", throwable.getMessage(), inBytes));
                return Option.apply(null); // skip the element
            }
    );
```

Reply:
@RayRoestenburg so no ProtoOutlet.create?

Reply:
Only noticed this now, yes we should add that too to be consistent.

Reply:
There is a ProtoInlet.create that takes a `(Array[Byte], Throwable) => Optional[T]`

Reply:
I'm totally lost on this. I don't see the API you're referring to in `current`. These are the two `create`s I can see in the API:
```
static <T extends com.google.protobuf.GeneratedMessageV3>ProtoInlet<T> | create(java.lang.String name, java.lang.Class<T> clazz)
-- | --
static <T extends com.google.protobuf.GeneratedMessageV3>ProtoInlet<T> | create(java.lang.String name, java.lang.Class<T> clazz, boolean hasUniqueGroupId)
```
So without the `.withErrorHandler` I'm at a lost. Sorry.

Reply:
Please see https://github.com/lightbend/cloudflow/blob/9d24f6edc36cf8e21a810d1b1061ba4aa5793cf6/core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/proto/javadsl/ProtoInlet.scala#L48